frontend/src/components/Pages/RobotPage/RobotPage.tsx

```diff
@@ -84,7 +86,14 @@ export function RobotPage() {
                             battery={selectedRobot?.batteryLevel}
                             batteryStatus={BatteryStatus.Normal}
                         />
-                        <PressureStatusView itemSize={48} pressure={selectedRobot?.pressureLevel} />
+                        {selectedRobot && (
```

Comment:
Inconsistent with the rest of the function since everything  is dependent on selectedRobot.

"selectedRobot &&" should either be removed or it should wrap around everything that depends on selectedRobot

Reply:
I would try to wrap around everything, as we should not render anything on a specific robot page unless we have a selected robot in my understanding.